src/app/app.module.ts

```diff
@@ -27,6 +32,7 @@ import { About4dvdComponent } from "./about4dvd.component";
 import { AppComponent } from "./app.component";
 import { ColorMapMenuComponent } from "./color-map-menu.component";
 import { DatasetMenuComponent } from "./dataset-menu.component";
```

Comment:
Why is the old component still here? We should totally do away with it.

Reply:
Removed the old component